pennylane/ops/cv.py

```diff
@@ -243,6 +253,11 @@ def _heisenberg_rep(p):
         U[3:5, 1:3] = s * R
         return U
 
+    def adjoint(self, do_queue=False):
+        return Beamsplitter(
+            -self.parameters[0], -self.parameters[1], wires=self.wires, do_queue=do_queue
```

Comment:
Is this correct? I could have sworn is was just the first parameter that should be negated. (@nquesada?)

Reply:
I'll just take it out for now.